mailchimp-marketing/1.0/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,43 @@
+= Mailchimp Marketing Connector - Mule 4
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for Mailchimp Marketing (Mailchimp Marketing Connector) enables you to integrate your applications with the Mailchimp Marketing data and functionality. This connector exposes operations provided by the Mailchimp Marketing API.
+
+For information about compatibility and fixed issues, refer to the Mailchimp Marketing Connector Release Notes.
+
+== Prerequisites
+
+To use this connector, you must be familiar with:
+
+* The Mailchimp Marketing Connector’s API
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+* Elements and global elements in a Mule flow
+* How to create a Mule app using Anypoint Studio
+
+Before creating an app, you must have:
+
+* Credentials to access the Mailchimp Marketing target resource
+* Anypoint Platform
+* Anypoint Studio version 7.3.5 or later
+
+== Common Use Cases for the Connector
+
+Some common use cases for Mailchimp Marketing Connector include:
+
+*
```

Comment:
* Creating and sending campaigns
* Adding, updating, and removing subscribers from lists
* Monitoring subscribers/unsubscribers counts and campaign email open rates